index.js

```diff
@@ -80,6 +119,10 @@ module.exports = function (src, dest, opts) {
 			});
 		}
 	});
+
+	promise.events = progress.emitter;
```

Comment:
As far as I understand, @sindresorhus suggested sth. like (https://github.com/sindresorhus/cpy/issues/33#issuecomment-232634292):
```js
promise.on = progress.emitter.on.bind(progress.emitter);
```

So, one can use it that way:
```js
const p = cpFile(…);
p.on(…);
p.then(…)
```